### DISCUSSION TITLE
The dependency parse result is different between displacy and local environment.

### DISCUSSION THREAD

**Discussion by meteorlin** [explosion/spaCy#10855] [Help: Coding & Implementations] [answered]
Hello!
I have installed spaCy (3.3.0) and used "en_core_web_??" (3.3.0) in my local environment. 
![image](https://user-images.githubusercontent.com/49402403/170413471-4ff5d67b-6792-464c-9ceb-aaead891ea5c.png)
But when I test the text of "Save Mary Jane" under the model setting of "en_core_web_sm" or "en_core_web_md", the dependency parser give me the wrong result which mistake "Save" as "PROPN". 
```python
import spacy
nlp = spacy.load("en_core_web_sm")  # or "en_core_web_md"
doc = nlp("Save Mary Jane")
spacy.displacy.serve(doc, style="dep")
```
![image](https://user-images.githubusercontent.com/49402403/170413575-319fb8a8-56c8-4387-8522-88f573ebf531.png)
For comparison, I test it in online [displacy demo](https://explosion.ai/demos/displacy?text=Save%20Mary%20Jane&model=en_core_web_sm&cpu=0&cph=0), but it give me the right answer under the model setting of "en_core_web_sm".
![image](https://user-images.githubusercontent.com/49402403/170413424-716bf6cc-5175-4754-b0f7-6a35ab1870e2.png)
After these experiments, I try "en_core_web_lg" in local environment and it gave me the same answer as online displacy demo gave me. 
![image](https://user-images.githubusercontent.com/49402403/170413634-a4c429f9-b1a1-4017-a6f8-5240693a14cc.png)
So, is "en_core_web_lg" model used in the online displacy demo? Or just my settings are wrong or miss other important settings in my local environment?
Thanks in advance!

**Comment by polm** [comment, marked as answer]
As visible in your screenshot, the online demo is using 3.1.0, which is why it's different from your local 3.3.0 output.

**Reply by meteorlin** [reply]
Oh! I reinstall the spaCy and en_core_web_sm in 3.1.0 version and it works fine. But I am so curious that why the latest model will perform worse than the previous?

**Reply by polm** [reply]
As noted in #3052, the models are statistical and explaining individual predictions is difficult. In this case it's likely the model is "worse" in this particular case but better in others.
